test/feature/user_action/user_action_delete_test.dart

```diff
@@ -61,24 +80,3 @@ void main() {
     expect((successAppState.userActionListState as UserActionListSuccessState).userActions.length, 2);
```

Comment:
ajouter expect(successAppState. userActionDeleteState is UserActionDeleteFailureState, isTrue);